src/pages/MetricDetail.svelte

```diff
@@ -204,6 +207,14 @@
       <td>Version</td>
       <td>{metric.version || 0}</td>
     </tr>
+    {#if searchfoxMap[params.app]}
+      <tr>
+        <td>Find in</td>
+        <td>
+          <SearchfoxLink metricName={metric.name} appName={params.app} />
```

Comment:
Creating a component here seems like overkill to me. How about just extracting out the getSearchFoxLink method in the component and put into `src/formatters/searchfox.js`? Bonus: in that case you can skip importing `searchFoxMap` in this file, and just check if `getSearchFoxLink` returns a value.